From 249057b7afee1ec16bc06451f6a1fd8f40d23200 Mon Sep 17 00:00:00 2001
From: Paolo Bonzini <pbonzini@redhat.com>
Date: Wed, 8 Jun 2016 16:40:40 +0200
Subject: [PATCH 2/6] scsi: mark TYPE_SCSI_DISK_BASE as abstract

RH-Author: Paolo Bonzini <pbonzini@redhat.com>
Message-id: <1465404041-26965-2-git-send-email-pbonzini@redhat.com>
Patchwork-id: 70572
O-Subject: [RHEL7.3 qemu-kvm-rhev PATCH 1/2] scsi: mark TYPE_SCSI_DISK_BASE as abstract
Bugzilla: 1338043
RH-Acked-by: Auger Eric <eric.auger@redhat.com>
RH-Acked-by: Fam Zheng <famz@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>

Suggested-by: Stefan Hajnoczi <stefanha@redhat.com>
Reviewed-by: Stefan Hajnoczi <stefanha@redhat.com>
Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>
(cherry picked from commit 6214a11ac167b1e866c04a81360286d186f04d82)
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hw/scsi/scsi-disk.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hw/scsi/scsi-disk.c b/hw/scsi/scsi-disk.c
index 9824129..a00f8d1 100644
--- a/hw/scsi/scsi-disk.c
+++ b/hw/scsi/scsi-disk.c
@@ -2850,6 +2850,7 @@ static const TypeInfo scsi_disk_base_info = {
     .class_init    = scsi_disk_base_class_initfn,
     .instance_size = sizeof(SCSIDiskState),
     .class_size    = sizeof(SCSIDiskClass),
+    .abstract      = true,
 };
 
 #define DEFINE_SCSI_DISK_PROPERTIES()                                \
-- 
1.8.3.1

